src/core/src/op/fake_convert.cpp

```diff
@@ -61,15 +91,39 @@ bool FakeConvert::visit_attributes(ov::AttributeVisitor& visitor) {
 }
 
 void FakeConvert::validate_type() const {
-    const auto& valid_types = fake_convert::get_valid_types();
+    const auto& valid_types = fake_convert_details::get_valid_types();
     OPENVINO_ASSERT(std::find(valid_types.begin(), valid_types.end(), m_destination_type) != valid_types.end(),
                     "Bad format for f8 conversion type: " + m_destination_type);
 }
 
 bool FakeConvert::has_evaluate() const {
-    return false;
+    OV_OP_SCOPE(v13_FakeConvert_has_evaluate);
+    switch (get_input_element_type(0)) {
+    case element::f16:
+    case element::bf16:
+    case element::f32:
+        return true;
+    default:
+        return false;
+    }
 }
 
+bool FakeConvert::evaluate(ov::TensorVector& outputs, const ov::TensorVector& inputs) const {
+    OV_OP_SCOPE(v13_FakeConvert_evaluate);
+
+    OPENVINO_ASSERT(outputs.size() == 1);
+    OPENVINO_ASSERT(inputs.size() == 2 || inputs.size() == 3);
```

Comment:
It would good to add `shape_infer` as it looks like only set output shape from input is not correct and other shapes inputs 1,2, should be checked.

Reply:
Setting data input shape as output shape is correct, input validation/shape_infer is planned to be added within the ticket 124921.